### PR TITLE
Minify CSS

### PR DESCRIPTION
**Pull request by microsoftsam**
Minify CSS using postcss (cssnano)
 - adding autoprefixer
 - and rtlcss for blockly.css

### CODE REVIEW

**Comment by pelikhan** [comment]
Can we update our supported browser docs with those version numbers?

**Comment by microsoftsam** [comment]
@pelikhan They are already in: 
https://makecode.microbit.org/browsers/technical
or did you mean somewhere else?

**Comment by pelikhan** [comment]
Are they supposed to match? They don't.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by microsoftsam** [comment]
They do match.. Could you be more specific?

**Comment by pelikhan** [comment]
Oh right, looks like Chrome/Firefox Android are missing from the docs.

**Comment by microsoftsam** [comment]
Okay will add those. Thanks

**Comment by microsoftsam** [comment]
Added docs

File: package.json [hunk 1]
```diff
@@ -75,7 +75,10 @@
     "marked": "^0.3.5",
     "fuse.js": "2.6.1",
     "rimraf": "2.5.4",
-    "request": "^2.79.0"
+    "request": "^2.79.0",
+    "postcss": "^6.0.9",
+    "rtlcss": "^2.2.0",
+    "cssnano": "^3.10.0"
```
Comment (pelikhan) [inline package.json]: is the "3rd party license " file update accordingly?

Comment (microsoftsam) [inline package.json]: Yeah we need to re-run the script. @abchatra

**Pull request merged**